src/script/components/Avatar.tsx

```diff
@@ -144,7 +154,7 @@ const Avatar: FC<AvatarProps> = ({
       avatarSize={avatarSize}
       noBadge={noBadge}
       noFilter={noFilter}
-      onClick={clickHandler}
+      handleAvatarInteraction={handleAvatarInteraction}
```

Comment:
```suggestion
      onAvatarInteraction={handleAvatarInteraction}
```

The component should not use imperative style for callbacks. It doesn't really know what those callbacks actually do, so we could use the `on` convention (`onClick`, `onKeyboard`, `onAvatarInteraction` ...)